sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -383,7 +400,38 @@ bool _parse_options(int msg_type, gnrc_rpl_instance_t *inst, gnrc_rpl_opt_t *opt
                 }
 
                 break;
+            case (GNRC_RPL_OPT_SOLICITED_INFO):
+                DEBUG("RPL: RPL SOLICITED INFO option parsed\n");
```

Comment:
you should add this to the `included_opts` bitfield so that the caller knows which options are present

Reply:
done.